pkg/cloud/services/elb/eip.go

```diff
@@ -0,0 +1,50 @@
+package elb
+
+import (
+	"fmt"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/elbv2"
+
+	infrav1 "sigs.k8s.io/cluster-api-provider-aws/v2/api/v1beta2"
+)
+
+func getElasticIPRoleName() string {
+	return fmt.Sprintf("lb-%s", infrav1.APIServerRoleTagValue)
+}
+
+func (s *Service) allocatedPublicIpv4AddressFromByoIPPool(input *elbv2.CreateLoadBalancerInput) error {
```

Comment:
I think this should be present tense as we are performing the action with this one
```suggestion
func (s *Service) allocatePublicIpv4AddressFromByoIPPool(input *elbv2.CreateLoadBalancerInput) error {
```